SDK v2 migration guide.md

```diff
@@ -61,14 +61,22 @@ TODO: list breaking changes
 
 #### RegistryManager
 
-| V1 class#method | Changed? | Equivalent V2 class#method |
```

Comment:
As discussed, removing this column. Please remove it in other tables as you edit them.